Workspace/src/arm_interface/src/arm_motor_commands_node.cpp

```diff
@@ -0,0 +1,110 @@
+#include <ros/ros.h>
+#include <std_msgs/String.h> 
+#include <std_msgs/Int32.h>
+#include <stdbool.h>
+#include <list>
+
+struct msg_from_topics {
+	bool ik_status; 
+	std::list<float> data_points; 
+};
+
+class InitObjects {
+	private:
+	ros::init(argc, argv, "arm_motor_commands");
+	ros::NodeHandle nh;
+	ros::Publisher pub_user_to_GUI =  nh.advertise<std::list<float> >("GUI", 1000); 
```

Comment:
As I said in the last review, the message type is std_msgs::Float64Array, not std::list